wca-states.md

```diff
@@ -3,10 +3,7 @@
 <version>Version: June 19, 2017
 
 
-The WCA recognizes a total of 199 independent countries.
-
-
-## <states-list>UN Member States:
```

Comment:
Yeah I don't know how coding works :P I will add that back in, it makes sense!

Reply:
Ok, sorry, nevermind! It looks like the compiler expects you to include the `##` as well. (relevant code here: https://github.com/thewca/wca-regulations-compiler/blob/6b6aee02de716bb2c7a1b1626200b61bcc364bee/wrc/parse/lexer.py#L77). I think to get the build passing, you'll have to change this to:

```
## <states-list> The WCA recognizes a total of 199 countries.
```

(and I'd consider changing that period at the end to a colon)

It's also worth noting that adding the `##` will change how this looks over on https://www.worldcubeassociation.org/regulations/countries/ (the sentence will look like a heading), which may be undesirable.

@viroulep because he's the expert on this.